lib/internal/Magento/Framework/Backup/AbstractBackup.php

```diff
@@ -69,6 +72,13 @@
     protected $_lastErrorMessage;
 
     /**
+     * Keep Source files in Backup
+     *
+     * @var boolean
+     */
+    protected $keepSourceFile;
```

Comment:
We recommend that by default new properties are private.

Reply:
Also wondering if we should have a default value here to make it a bit more explicit that it is a boolean. I guess the php will behave the same way but might be nice to add in the default.